components/AsideBlock/AsideBlock.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { AsideBlockWrapper } from './style';
+import AppButton from '../AppButton/AppButton';
+
+const AsideBlock: React.FC = () => {
+  const [activeButton, setActiveButton] = useState('');
+
+  return (
+    <AsideBlockWrapper>
+      <AppButton
```

Comment:
you can pass a config to the component and it will render all the buttons you need